src/test/kotlin/com/ampnet/web3provider/service/AccountInformationServiceTest.kt

```diff
@@ -0,0 +1,79 @@
+package com.ampnet.web3provider.service
+
+import com.ampnet.web3provider.TestBase
+import com.ampnet.web3provider.controller.pojo.JsonRpcRequest
+import com.ampnet.web3provider.controller.pojo.ProviderResponse
+import com.ampnet.web3provider.enums.RedisEntity
+import com.ampnet.web3provider.repository.RedisRepository
+import com.ampnet.web3provider.service.impl.AccountInformationServiceImpl
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.Mockito
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.data.redis.core.RedisTemplate
+import org.springframework.restdocs.RestDocumentationContextProvider
+import org.springframework.restdocs.RestDocumentationExtension
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import org.springframework.web.context.WebApplicationContext
+import org.web3j.protocol.core.DefaultBlockParameterName
+import org.web3j.utils.Numeric
+import java.math.BigInteger
+
+@ExtendWith(value = [SpringExtension::class, RestDocumentationExtension::class])
+@SpringBootTest
```

Comment:
Is `SpringBootTest` really needed, can service test use just `DataJpaTest`?

Reply:
It doesn't work with @DataJpaTest, none of the classes implement JpaRepository or CrudRepository so there would be no usage of just the @DataJpatest annotation. I could manually import everything: 
`@Import(RedisTemplate::class, DefaultProviderService::class, RedisRepository::class, RedisConfig::class)` but in that case it fails to load application context because it cannot find some beans required for the redis template. I would leave it as is with the @SpringBootTest.

Reply:
I would rather use `Import` because it shortens the test speed dramatically.